manual/sphinx/user_docs/coordinates.rst

```diff
@@ -65,76 +68,109 @@ and the magnitudes of the reciprocal vectors are therefore
-i.e. the sign of the poloidal field. The new coordinates
-`\left(x,y,z\right)` are defined by:
+structures, the standard coordinate system used by BOUT++ models is a
+Clebsch system where grid-points are aligned to the magnetic field
+along the `y` coordinate.
+
+To align to the magnetic field we define a local field line pitch `\nu`:
 
 .. math::
-   :label: eq:coordtransform
 
    \begin{aligned}
-   x = {\sigma_{B\theta}}\left(\psi - \psi_0\right) \qquad y = \theta \qquad z = \sigma_{B\theta}
-   \left(\zeta - \int_{\theta_0}^{\theta}\nu\left(\psi,\theta\right)d\theta\right)
-   \end{aligned}
+   \nu\left(\psi, \theta\right) = \frac{{\boldsymbol{B}}\cdot\nabla\phi}{{\boldsymbol{B}}\cdot\nabla\theta} =
+   \frac{{B_{\text{tor}}}{h_\theta}}{{B_{\text{pol}}}R}
+
+The sign of the poloidal field `{B_{\text{pol}}}` and toroidal field 
+`{B_{\text{tor}}}` can be either + or -.
 
-Where `\nu` is the local field-line pitch given by
+The field-aligned coordinates `\left(x,y,z\right)` are defined by:
 
 .. math::
+   :label: eq:coordtransform
 
    \begin{aligned}
-   \nu\left(\psi, \theta\right) = \frac{{\boldsymbol{B}}\cdot\nabla\zeta}{{\boldsymbol{B}}\cdot\nabla\theta} =
-   \frac{{B_{\text{tor}}}{h_\theta}}{{B_{\text{pol}}}R} = \frac{\left(F/R\right)h_\theta}{{B_{\text{pol}}}R} = FJ/R^2\end{aligned}
-
-where `F={B_{\text{tor}}}R` is a function only of `\psi`
-(sometimes called the poloidal current function).
+   x = {\sigma_{B\theta}}\left(\psi - \psi_0\right) \qquad y = \theta \qquad z = \sigma_{B\theta}
+   \left(\phi - \int_{\theta_0}^{\theta}\nu\left(\psi,\theta\right)d\theta\right)
+   \end{aligned}
 
 The coordinate system is chosen so that `x` increases radially
-outwards, from plasma to the wall. The sign of the toroidal field
-`{B_{\text{tor}}}` can then be either + or -.
+outwards, from plasma to the wall. The `y` coordinate increases in the
+same direction as `\theta` i.e. clockwise in the right-hand poloidal
+plane. The `z` coordinate increases in the same direction as `\zeta`
+i.e.  anti-clockwise looking from the top if `B_{pol}>0` and clockwise
+if `B_{pol} < 0`.
+
+This coordinate system is right-handed if `B_{pol}>0`, and left-handed if `B_{pol}<0`. 
 
-The contravariant basis vectors are therefore
+The reciprocal basis vectors are therefore
 
 .. math::
 
    \begin{aligned}
-   \nabla x = {\sigma_{B\theta}}\nabla \psi \qquad \nabla y = \nabla \theta \qquad \nabla z =
-   {\sigma_{B\theta}}\left(\nabla\zeta - \left[\int_{\theta_0}^\theta{\frac{\partial \nu\left(\psi,
-   \theta\right)}{\partial \psi}} d\theta\right] \nabla\psi - \nu\left(\psi, \theta\right)\nabla\theta\right)\end{aligned}
-
+   \nabla x = {\sigma_{B\theta}}\nabla \psi \qquad
+   \nabla y = \nabla \theta \qquad
+   \nabla z = \nabla\zeta - \sigma_{B\theta}\left[\int_{\theta_0}^\theta{\frac{\partial \nu\left(\psi,\theta\right)}{\partial \psi}} d\theta\right] \nabla\psi
+   - \sigma_{B\theta}\nu\left(\psi, \theta\right)\nabla\theta\right)
```

Comment:
I think this closing bracket is superfluous now? Either that, or there's a missing `\left(` somewhere

```suggestion
   - \sigma_{B\theta}\nu\left(\psi, \theta\right)\nabla\theta
```